src/Confluent.Kafka/Impl/SafeKafkaHandle.cs

```diff
@@ -258,9 +258,9 @@ internal WatermarkOffsets GetWatermarkOffsets(string topic, int partition)
             return new WatermarkOffsets(low, high);
         }
 
-        internal void Subscribe(ICollection<string> topics)
+        internal void Subscribe(IEnumerable<string> topics)
         {
-            IntPtr list = LibRdKafka.topic_partition_list_new((IntPtr) topics.Count);
+            IntPtr list = LibRdKafka.topic_partition_list_new((IntPtr) topics.Count());
```

Comment:
yeah - you just needed to remind me `IEnumerable` is prevalent with LINQ. Anyway, this PR should be good without changes, hopefully merge on the weekend.